src/pages/storage/StoragePoolClusterMember.tsx

```diff
@@ -0,0 +1,38 @@
+import ResourceLink from "components/ResourceLink";
+import { useClusterMembers } from "context/useClusterMembers";
+import type { FC } from "react";
+import type { LxdStoragePool } from "types/storage";
+import { hasPoolMemberSpecificSize } from "util/storagePool";
+
+interface Props {
+  pool: LxdStoragePool;
+}
+
+export const StoragePoolClusterMember: FC<Props> = ({ pool }) => {
+  const hasMemberSpecificSize = hasPoolMemberSpecificSize(pool.driver);
+  const { data: clusterMembers = [] } = useClusterMembers();
+
+  return (
+    <div>
+      {hasMemberSpecificSize ? (
+        clusterMembers.map((member) => {
+          return (
+            <div className="clustered-resource-link" key={member.server_name}>
+              <ResourceLink
+                to="/ui/cluster"
+                type={"cluster-member"}
```

Comment:
```suggestion
                type="cluster-member"
```
We should avoid the `{` where not needed. Same applies below in the file.

Reply:
This seems not applied yet.